<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PSK31TXSettingsDialog</class>
 <widget class="QDialog" name="PSK31TXSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>554</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Transmit Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="listControls">
     <property name="title">
      <string>Text</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="prefixCRLF">
        <property name="toolTip">
         <string>Prefix text with carriage returns and line feed</string>
        </property>
        <property name="text">
         <string>Prefix CR+CR+LF</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Predefined text</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="add">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add item to list</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="remove">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove selected items from list</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="up">
          <property name="toolTip">
           <string>Move selected item up</string>
          </property>
          <property name="text">
           <string>Up</string>
          </property>
          <property name="icon">
           <iconset resource="../../../sdrgui/resources/res.qrc">
            <normaloff>:/arrow_up.png</normaloff>:/arrow_up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="down">
          <property name="toolTip">
           <string>Move selected item down</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../sdrgui/resources/res.qrc">
            <normaloff>:/arrow_down.png</normaloff>:/arrow_down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="postfixCRLF">
        <property name="toolTip">
         <string>Postfix text with carriage returns and line feeds</string>
        </property>
        <property name="text">
         <string>Postfix CR+CR+LF</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QListWidget" name="predefinedText">
        <property name="toolTip">
         <string>Predefined text messages

Substitutions:
${callsign}
${location}</string>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fskGroup">
     <property name="title">
      <string>Modulation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lpfTapsLabel">
        <property name="text">
         <string>RF BW limit LPF taps</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="lpfTaps">
        <property name="toolTip">
         <string>Number of taps in LPF for RF BW filter.</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="pulseShaping">
        <property name="toolTip">
         <string>Enable raised cosine pulse shaping filter</string>
        </property>
        <property name="text">
         <string>Raised cosine pulse shaping</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="betaLabel">
        <property name="text">
         <string>Filter rolloff (beta)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="beta">
        <property name="toolTip">
         <string>Roll-off of the filter</string>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="symbolSpanLabel">
        <property name="text">
         <string>Filter symbol span</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="symbolSpan">
        <property name="toolTip">
         <string>Number of symbols over which filter is applied</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="debugGroup">
     <property name="title">
      <string>Debug</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="rfNoise">
        <property name="toolTip">
         <string>Generate white noise as RF signal.</string>
        </property>
        <property name="text">
         <string>Generate RF noise</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PSK31TXSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PSK31TXSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
